test/AsmResolver.Workspaces.Tests/WorkspaceIndexNodeDataTest.cs

```diff
@@ -0,0 +1,245 @@
+using AsmResolver.Workspaces.Tests.Mock;
```

Comment:
Quite a few inconsistencies with regards to formatting, spacing and use of `var` instead of `int` in this file. Have seen this in your previous PRs as well. Make sure you use the configuration as specified in the `.editorconfig`.